lib/features/mailbox_dashboard/presentation/model/search/search_email_filter.dart

```diff
@@ -57,6 +61,7 @@ class SearchEmailFilter with EquatableMixin {
     Option<UTCDate>? beforeOption,
     Option<UTCDate>? startDateOption,
     Option<UTCDate>? endDateOption,
+    Set<Comparator>? sortOrder,
```

Comment:
should we use `Option` for it?

Reply:
I checked and `Option` is a better solution so i changed it. Thank you for the suggestion!

Reply:
why you think that?

Reply:
Cause the sort type `Relevance` means no sorting so i though that using `Option` with return None() is better than return null or `<Comparator>{}`.